Packs/CTM360-CyberBlindspot/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,48 @@
+
+#### Dashboards
+
+##### CyberBlindspot Incident Dashboard
+
+- Updated dashboard widget sizes.
+- Added more widgets.
+
+
+#### Incident Fields
+
+##### New: CTM360 CyberBlindspot Module
+
+New: Incident field to identify which module the incident belongs to.
+(Available from Cortex XSIAM 2.0).
+(Available from Cortex XSOAR 6.10.0).
+
+#### Incident Types
+
+##### CyberBlindspot Incident
+
+Updated CyberBlindspot incidents to use V2 playbook.
+
+#### Integrations
+
+##### CTM360 CyberBlindspot
+
+- Update pack **README** file.
+- Added module type configuration (defaults to 'Incidents'):
+  - Incidents
+  - Compromised Cards
+  - Breached Credentials
+  - Domain Infringement
+  - Subdomain Infringement
+
+#### Mappers
+
+##### CTM360-CyberBlindspot - Incoming Mapper
+
+Added mapping for the new incident field "CTM360 CyberBlindspot Module"
+
+#### Playbooks
+
+##### New: CyberBlindspot Incident Management V2
+
+This playbook runs the incidents through indicator enrichment, then based on the mirroring settings, it can communicate with the remote server to assist the user in the next course of action (Whether it be closing the incident, initiating the takedown of an online asset or simply waiting for a process on the remote server to end) to take on the incident if any. The 2nd iteration.
```

Comment:
```suggestion
This playbook runs the incidents through indicator enrichment, then based on the mirroring settings, it can communicate with the remote server to assist the user in the next course of action (Whether it be closing the incident, initiating the takedown of an online asset or simply waiting for a process on the remote server to end) to take on the incident if any. 
```